src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,83 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+    Sign sign;
+
+    public int calculate(String rpnString){
+        int count = 0;
+        String[] retval = rpnString.split(" ");
+
+         while (retval.length > count){
+            if(retval[count].matches("^[0-9]+$")){
+                numbers.push(Integer.parseInt(retval[count]));
+            } else {
```

Comment:
You have here 3 floors of logical operations. Swithc inside If inside While. This is very bad, you should have ideally no more than one, as an exception 2 floors. Extract Swithc operator into some private method. 
PS : Pay attention on it ! This is one of the principles of clean code and refactoring remember it forever! No christmas trees in method